hedera-node/hedera-mono-service/src/main/java/com/hedera/services/txns/contract/ContractCallTransitionLogic.java

```diff
@@ -104,16 +107,28 @@ public void doStateTransitionOperation(
             final long maxGasAllowanceInTinybars,
             final BigInteger offeredGasPrice) {
         var op = contractCallTxn.getContractCall();
-        final var target = targetOf(op);
-        final var targetId = target.toId();
 
         // --- Load the model objects ---
         final var sender = accountStore.loadAccount(senderId);
 
-        Account receiver =
-                entityAccess.isTokenAccount(targetId.asEvmAddress())
-                        ? new Account(targetId)
-                        : accountStore.loadContract(targetId);
+        final var target = targetOf(op);
```

Comment:
Does adding this mean we automatically start supporting ContractCallTransactions with alias targets?

Reply:
Negative. If the target is a non-existing alias, we only let it pass [if it is an EthereumTransaction,  the EVM version is not 0.30 and all the needed flags for lazy creation are enabled.](https://github.com/hashgraph/hedera-services/blob/fefe3534d1555c5b8052f6d16af5fc5c27bf0607/hedera-node/hedera-mono-service/src/main/java/com/hedera/services/txns/contract/ContractCallTransitionLogic.java#L117-L121). If any of these conditions are not present, we just fall back to the [previous handling](https://github.com/hashgraph/hedera-services/blob/fefe3534d1555c5b8052f6d16af5fc5c27bf0607/hedera-node/hedera-mono-service/src/main/java/com/hedera/services/txns/contract/ContractCallTransitionLogic.java#L127-L129).